hub/management/commands/import_mps_relevant_votes.py

```diff
@@ -104,10 +138,14 @@ def create_data_types(self, votes, edms):
         vote_options = [
             {"title": "Aye", "shader": "#89c489"},
             {"title": "No", "shader": "#e18674"},
-            {"title": "No vote", "shader": "#fede86"},
+            {"title": "Did not vote", "shader": "#fede86"},
+            {"title": "Did not vote (teller)", "shader": "#cff4fc"},
+            {"title": "Not in office", "shader": "#073a41"},
```

Comment:
We should probably get @zarino to suggest some non red/green colours here for accessibility reasons.

Reply:
Thank you! :D